sencha-workspace/SlateDemonstrationsTeacher/app/controller/Dashboard.js

```diff
@@ -47,10 +47,41 @@ Ext.define('SlateDemonstrationsTeacher.controller.Dashboard', {
     ],
 
     refs: {
-        dashboardCt: 'slate-demonstrations-teacher-dashboard',
-
         studentProgressGrid: 'slate-demonstrations-teacher-dashboard slate-demonstrations-teacher-studentsprogressgrid',
-        teacherOverviewwindow: 'slate-demonstrations-teacher-skill-overviewwindow'
+        teacherOverviewwindow: 'slate-demonstrations-teacher-skill-overviewwindow',
```

Comment:
@aliyeysides you should not define refs only for sake of using the shorter string in the `control` config. Only define a ref when you need to access the instance from one of the controllers methods -- then you may replace any identical selector under `control` with the ref name.

Further, it is not valid to use a ref for OverviewWindow because OverviewWindow is not a singleton in the context of the application, there can be any-number of windows open at a given time within the application. A string selector in `control` will catch events from any matching instance whereas a ref would cache an arbitrary one of the set of windows.